lib/trailblazer/operation.rb

```diff
@@ -66,6 +80,16 @@ def run(*params)
 
       [process(*params), valid?].reverse
     end
+    
+    def fetch_collection(*params)
+      setup_collection!(*params)
+      
+      @collection = fetch(*params)
+      @search = perform_search(*params) if self.respond_to?(:perform_search)
```

Comment:
Sorry bro, but we can't have `respond_to` in Trailblazer. I :heart: the idea of adding generic pagination support to operations, but this has to be implemented with a module that overrides this method or hooks into the workflow in some other way.

Great idea!!!